packages/components/psammead-most-read/src/list.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import styled from 'styled-components';
+import { shape, string, arrayOf } from 'prop-types';
+import {
+  Burmese,
+  Bengali,
+  EasternArabic,
+  WesternArabic,
+} from '@bbc/psammead-locales/numerals';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import Grid from '../../psammead-grid';
+import { MostReadRank, MostReadLink } from './item';
+
+const MostReadItemProps = {
+  columns: {
+    group0: 5,
+    group1: 5,
+    group2: 5,
+    group3: 5,
+    group4: 5,
+    group5: 5,
+  },
+};
+
+const MostReadRankProps = {
+  item: true,
+  columns: {
+    group0: 1,
+    group1: 1,
+    group2: 1,
+    group3: 1,
+    group4: 1,
+    group5: 1,
+  },
+};
+
+const MostReadLinkProps = {
+  item: true,
+  columns: {
+    group0: 4,
+    group1: 4,
+    group2: 4,
+    group3: 4,
+    group4: 4,
+    group5: 4,
+  },
+};
+
+const MostReadListProps = {
+  enableGelGutters: true,
+  enableGelMargins: true,
+  columns: {
+    group0: 1,
+    group1: 1,
+    group2: 2,
+    group3: 2,
+    group4: 2,
+    group5: 5,
+  },
+};
+
+const serviceNumerals = service => {
+  const servicesNonWesternNumerals = {
+    arabic: EasternArabic,
+    bengali: Bengali,
+    burmese: Burmese,
+    pashto: EasternArabic,
+    persian: EasternArabic,
+    urdu: EasternArabic,
+  };
+  return servicesNonWesternNumerals[service]
+    ? servicesNonWesternNumerals[service]
+    : WesternArabic;
+};
+
+const StyledOl = styled.ol.attrs({
+  role: 'list',
+})`
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+`;
+
+const MostReadItem = ({ service, script, item, rank, dir }) => (
+  <li>
+    <Grid {...MostReadItemProps} dir={dir}>
```

Comment:
You can make this outer Grid change its base element and be an `li` instead of a `div` by adding the `forwardedAs` prop like this:
```suggestion
    <Grid {...MostReadItemProps} dir={dir} forwardedAs="li">
```
https://www.styled-components.com/docs/api#forwardedas-prop
This may also fix the issue of the Grid fallback not working, since the outer `<Grid>` on the MostReadList will be a direct parent of the `<Grid>` here in the MostReadItem.

Reply:
I had a play around with this for quite some time today and I couldn't seem to get it working. I tried just adding the `fowardedAs` prop without removing anything else, adding it with removing the `<li>` tag, adding it to the `MostReadList` too using `forwardedAs=ol`, etc.
I'm not sure if I just got unlucky with the permutations of the options I had played with but more often than not, it just seemed to break the version that works on chrome. I think this might need some further investigating.

Reply:
Currently being investigated in #2643